draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -233,6 +234,56 @@ The following example is the decoded header and payload of a JWT meeting the pro
 }
 ~~~
 
+# Authorization Server-provided nonces
+
+This section specifies a mechanism using server provided nonces to limit the lifetime of a Client Attestation PoP JWT, similar to the one defined in section 8 of {{RFC9449}}.
```

Comment:
Maybe add something along this line here:
A nonce value is a string as defined in (reference to nonce-syntax) that is opaque to the client.